cronner/__init__.py

```diff
@@ -1 +1,2 @@
 from .cronner import *
+from .kronjob_util import serialize_kronjob
```

Comment:
Have a slight preference for this not being a top-level export but not a big deal if you have a preference for this

Reply:
I guess i don't understand imports for python2.7 i changed to an explicit relative import in configure (and not a top-level export)